### PR TITLE
Add more information around ssh keys and add ref links.

### PR DESCRIPTION
**Pull request by stephenlauck**
I added some more information around why and how SSH keys work just to give some context to the instructions.

Signed-off-by: Stephen Lauck <stephen.lauck@gmail.com>

### CODE REVIEW

**Comment by NBS111** [comment]
Looking goo to me! :+1:

**Comment by dabogsta** [comment]
:shipit:

**Comment by VTVL-CODE** [comment]
Looks fine to me. 👍

**Comment by NBS111** [comment]
:1st_place_medal:

**Comment by thevinty** [comment]
Looks great!

**Review by Fendra-beep — COMMENTED** [review]
Looks Good 👍

**Comment by BRReed** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ Persistent settings
 
 ### SSH Keys
 
+Generating a pair of SSH keys allow you to ssh login to your server without typing or remembering your password. There is a private and public key. You keep your private key secret and safe and give out your public key to any server you would like to securely connect to. SSH keys are better than passwords because you cannot "forget" them and they are not easily figured out or comprimised.
```
Comment (JoshRodstein) [inline README.md]: comprimise -> compromise

**Comment by josephhernandez209** [comment]
nice 👍

File: README.md (hunk 1 above)
Comment (racheljoyrodas) [inline README.md]: compromised spelling change

Comment (stephenlauck) [reply README.md]: fixed you F'er.

Comment (stephenlauck) [reply README.md]: thank you for your review but maybe you should fix it yourself next time @racheljoyrodas .

**Pull request merged**